client/src/components/Signup/Signup.css

```diff
@@ -0,0 +1,15 @@
+ body{
+    background-image: url('c.jpg');
```

Comment:
please add the proper image name

Reply:
It would be great if you suggest some valid name.